Apps/W1/HybridGP/app/src/Migration/Items/GPItemMigrator.codeunit.al

```diff
@@ -370,4 +370,60 @@ codeunit 4019 "GP Item Migrator"
                 exit(CostingMethodOption::Standard);
         end;
     end;
+
+    procedure MigrateItemClasses()
+    var
+        GPCompanyAdditionalSettings: Record "GP Company Additional Settings";
+        GPIV00101: Record "GP IV00101";
+        GPIV40400: Record "GP IV40400";
+        InventoryPostingGroup: Record "Inventory Posting Group";
+        InventoryPostingSetup: Record "Inventory Posting Setup";
+        Item: Record Item;
+        HelperFunctions: Codeunit "Helper Functions";
+        ClassId: Text[11];
+        AccountNumber: Code[20];
+        MigrateItemClasses: Boolean;
+    begin
+        MigrateItemClasses := true;
+        if GPCompanyAdditionalSettings.Get(CompanyName()) then
+            MigrateItemClasses := GPCompanyAdditionalSettings."Migrate Item Classes";
+
+        if not MigrateItemClasses or not GPIV00101.FindSet() then
+            exit;
+
+        repeat
+            Clear(GPIV40400);
+            Clear(InventoryPostingGroup);
+            Clear(InventoryPostingSetup);
+
+            ClassId := GPIV00101.ITMCLSCD.Trim();
+            if ClassId <> '' then
+                if Item.Get(GPIV00101.ITEMNMBR) then begin
```

Comment:
GP's ITEMNMBR is much longer than what BC supports. Customers should have already 'fixed' those long numbers to fit into BC, but it is something to be aware of.